app/Jobs/ApproveArticle.php

```diff
@@ -18,6 +18,7 @@ public function __construct(Article $article)
     public function handle(): Article
     {
         $this->article->approved_at = Carbon::now();
+        $this->article->declined_at = null;
```

Comment:
Hmm, I don't think this can ever happen? Either an article is approved or declined. It seems weird to explicitly set the `declined_at` to `null` if it's never been filled out in the first place.

If this gave you trouble I suggest making this column a `DEFAULT NULL` in the migration.

Reply:
Ah yeah, the declined_at column is already null by default. I've removed setting it to null here. This would only have made sense with the undecline feature.